.github/workflows/nightly.yaml

```diff
@@ -43,7 +43,7 @@ jobs:
           version: v3.8.0-1
 
       - name: Run chart-testing (lint)
-        run: ct lint --all --config .github/ct-redpanda.yaml
+        run: ct lint --config .github/ct.yaml
```

Comment:
Don't we still need
```suggestion
        run: ct lint --config .github/ct-redpanda.yaml
```

Reply:
I was thinking that nightly should lint all charts. That's why I bring back ct.yaml configuration. WDYT?

Reply:
What is the intent of the nightly now? I thought it was to run tests only for the redpanda chart, after all we have a matrix. Should we have a nightly for all charts that just checks linting?

Reply:
reverted